dev/jobs/arch_tars.sh

```diff
@@ -4,7 +4,7 @@ set -x
 
 ###############################################################
 # Source FV3GFS workflow modules
-. "${HOMEgfs}"/ush/load_fv3gfs_modules.sh
+source "${HOMEgfs}"/ush/load_fv3gfs_modules.sh
```

Comment:
Missed /dev/ush here

Reply:
My `sed` magic loop didn't apply to `"${HOMEgfs}"/ush`.   Should be fixed now.  Thx.